docs/source/how_tos/installation.rst

```diff
@@ -45,13 +45,21 @@ Or from http:
 
 .. todo:: pipy.org/pip installation not available yet.
 
-Windows
--------
-To install Apache Spark on Windows, follow `these instructions <https://sparkbyexamples.com/pyspark-tutorial/#pyspark-installation>`_.
-
 
 Registry
 ========
+
+Standalone Registry
+-------------------
+To use dsgrid in your own computational environment, you will need to run ArangoDB, and either set
+up a new dsgrid registry or load a dsgrid registry that has been written to disk. 
+
+.. todo:: Link to how-to for setting up a standalone ArangoDB on laptops, etc.
+
+.. todo:: Link to how-to for setting up a standalone dsgrid registry on Eagle
+
+NREL Shared Registry
```

Comment:
I would list this first since it is the default behavior. New users will likely use it first.